websites/D/Dominion Online/presence.ts

```diff
@@ -0,0 +1,58 @@
+const presence = new Presence({
+  clientId: "849684658563055627"
+});
+
+let lobbyStartTime: number,
+  gameStartTime: number;
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    startTimestamp: Math.floor(Date.now() / 1000)
```

Comment:
Probaly just remove this?

Reply:
The startTimestamp is now preserved. (https://github.com/PreMiD/Presences/pull/3886/commits/9f133490ceea8a5a40d1b286376ee6cb9305e6e9)
It has different startTimestamps for lobby, game, and other cases.